lib/render/AnnotationRenderer.cpp

```diff
@@ -514,14 +514,6 @@ void AnnotationRenderer::InScenePaint(size_t ts)
 
 	vector<string> names = m_paramsMgr->GetDataMgrNames();
     Transform* t = vpParams->GetTransform(names[0]);
-    if (names.size()) {
-        Transform* tmp;
-        for (int i=0; i<names.size(); i++) {
-            tmp = vpParams->GetTransform(names[i]);
-            if ( tmp->GetScales()[Z] < t->GetScales()[Z] )
-                *t = *tmp;
-        }
-    }
```

Comment:
I believe this will reintroduce issue #1214 

In retrospect, I should have commented this code though.  It's taking the smallest z scale factor from all datasets, and using that is the default.  If a ROMS default scale factor is 1000, this will blow up the scene if it's loaded alongside of a WRF dataset.

Reply:
I don't quite understand that bug. It says "[if auto scaling is enabled ... step 1: make sure auto scaling is disabled]"

How would this fix that bug? The annotation renderer shouldn't fix the renderers' scaling factors?

Reply:
What this code is doing is 

1) Gather the default scale factors for all of the data managers
2) Select the smallest scale factor, and apply that to the ViewpointParams

Say we loaded an ocean model with a default scale factor of 1000, and an atmospheric model with a scale factor of 50. 

When we randomly take the scale factor associated with names[0], we could be applying the ocean model 1000x scale.  This makes the atmospheric domain too big, and unusable.  

Therefore we look at all scale factors, and use the smallest one for the ViewpointParams.

Reply:
@sgpearse you have approved this PR, but it's not clear that your concern regarding a regression has been addressed: "I believe this will reintroduce issue #1214". Has it been addressed?

Reply:
I went ahead tested this branch to see if #1214 had regressed, but I wasn't able to reproduce it.  Since there's no regression, I approved the PR.